frontend/src/router/routeHandlers.ts

```diff
@@ -30,24 +32,28 @@ import type { RouteLocationNormalized } from 'vue-router';
 
 const ACCESS_RESTRICTED_ERROR = new FamRouteError(
     RouteErrorName.ACCESS_RESTRICTED,
-    "Access restricted"
+    'Access restricted'
 );
 
 // --- beforeEnter Route Handler
 
 const beforeEnterDashboardRoute = async (to: RouteLocationNormalized) => {
     let applicationAdmins;
     let userRolesFetchResult;
+    let delegatedAdmins;
+
     if (selectedApplicationId.value === FAM_APPLICATION_ID) {
-        applicationAdmins = await asyncWrap(fetchApplicationAdmins())
+        applicationAdmins = await asyncWrap(fetchApplicationAdmins());
     } else {
         userRolesFetchResult = await asyncWrap(
             fetchUserRoleAssignments(selectedApplicationId.value)
         );
+        fetchDelegatedAdmins(selectedApplicationId.value);
```

Comment:
Hi Jota, you're making the call to fetch delegated admin, but not assign the return result to your delegatedAdmins variable....

Reply:
:sweat_smile: